request_logging/middleware.py

```diff
@@ -11,27 +11,32 @@ class LoggingMiddleware(MiddlewareMixin):
 
     def process_request(self, request):
         request_logger.info(colorize("{} {}".format(request.method, request.get_full_path()), fg="cyan"))
-        if (request.body):
-            self.log_body(self.chunked_to_max(request.body))
+        headers = {k: v for k, v in request.META.items() if k.startswith('HTTP')}
+        if headers:
+            self.log(headers)
+        if request.body:
+            self.log(self.chunked_to_max(request.body))
 
     def process_response(self, request, response):
         resp_log = "{} {} - {}".format(request.method, request.get_full_path(), response.status_code)
         if (response.status_code in range(400, 600)):
             request_logger.info(colorize(resp_log, fg="magenta"))
-            self.log_resp_body(response, level=logging.ERROR)
+            self.log_resp(response, level=logging.ERROR)
         else:
             request_logger.info(colorize(resp_log, fg="cyan"))
-            self.log_resp_body(response)
+            self.log_resp(response)
 
         return response
 
-    def log_resp_body(self, response, level=logging.DEBUG):
+    def log_resp(self, response, level=logging.DEBUG):
         if (not re.match('^application/json', response.get('Content-Type', ''), re.I)):  # only log content type: 'application/xxx'
             return
 
-        self.log_body(self.chunked_to_max(response.content), level)
+        self.log(response._headers, level)
+        self.log(self.chunked_to_max(response.content), level)
 
-    def log_body(self, msg, level=logging.DEBUG):
+    @staticmethod
```

Comment:
I'd make sure `log` and `log_resp` are consistent - either they are all `@staticmethod` or they are not.